Cargo.toml

```diff
@@ -20,6 +20,30 @@ opt-level = "z" # smaller optimizations
 [profile.dev]
 opt-level = 1 # no optimizations was just too painful in terms of flash size
 
+# Per-package profiles do not currently allow lto, panic, or rpath.
+# This costs us an extra 456 bytes in the endoscope blob that is injected
+# into the SP to measure its active flash bank.
```

Comment:
How close are we to hitting our limit? Do we really need to be concerned about this right now?

Reply:
I'll delete this. The same information is in endoscope's README.md and it's not a concern right now.